### PR TITLE
Fix fade material translucent, add fade polyline development sandcastle example

### PR DESCRIPTION
**Pull request by hpinkos**
Fade material wasn't setting translucent properly when the `fadeOutColor` was translucent.

I also added an example for how to use the fade material with a polyline to the `development/polylines` demo
(It doesn't seem possible to use this material in the Entity API since there is no corresponding `MaterialProperty`)

### CODE REVIEW

**Comment by pjcozzi** [comment]
LGTM, thanks @hpinkos!

@emackey do you want to look at this?

File: Apps/Sandcastle/gallery/development/Polylines.html [hunk 1]
```diff
@@ -99,6 +99,24 @@
         material : Cesium.Material.fromType(Cesium.Material.PolylineArrowType)
     });
     Sandcastle.declare(localPolyline); // For highlighting on mouseover in Sandcastle.
+    
+    //Polyline using the fade material
+    var fadingPolyline = polylines.add({
+        positions : Cesium.Cartesian3.fromDegreesArrayHeights([-75, 43, 500000,
```
Comment (emackey) [inline Apps/Sandcastle/gallery/development/Polylines.html]: Can you wrap this in `Cesium.PolylinePipeline.generateCartesianArc` so curves with the globe?  The other lines in this demo do that.

File: Apps/Sandcastle/gallery/development/Polylines.html [hunk 2]
```diff
@@ -99,6 +99,24 @@
...
+                                                               -125, 43, 500000]),
+        width : 5,
+        material : Cesium.Material.fromType(Cesium.Material.FadeType, {
+            repeat: false,
+            fadeInColor: Cesium.Color.BLUE,
```
Comment (emackey) [inline Apps/Sandcastle/gallery/development/Polylines.html]: &lt;bikeshedding&gt;`BLUE` is too dark, use `CYAN`&lt;/bikeshedding&gt;

File: Apps/Sandcastle/gallery/development/Polylines.html [hunk 3]
```diff
@@ -99,6 +99,24 @@
...
+            fadeOutColor: Cesium.Color.TRANSPARENT,
```
Comment (emackey) [inline Apps/Sandcastle/gallery/development/Polylines.html]: So `TRANSPARENT` is actually implicitly a transparent `BLACK` color.  Instead, use `Cesium.Color.CYAN.withAlpha(0)`.

@hpinkos I know from our logo work that you have a good eye for shades of color.  Check out the difference with and without this change.

**Comment by hpinkos** [comment]
@emackey thanks for the suggestions!  Updated

**Comment by hpinkos** [comment]
@emackey the half line length fade was happening because I was misusing the time property.  I pushed a fix to master https://github.com/AnalyticalGraphicsInc/cesium/commit/7d8ee10f86794e8009f834f4529a54c36442f298

**Pull request merged**